CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,89 @@
+# Changelog 1.13
+
+## 1.13.0 - 2023-07-24
+
+### Added
```

Comment:
Consider reordering items by importance and categories.

Reply:
Reordered